From 8f7f23b67dad8414f3dc0821b8307652af926510 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@freescale.com>
Date: Fri, 23 Jan 2015 09:58:43 +0800
Subject: [PATCH 1409/1543] MLK-10102-11 usb: chipidea: imx: export
 imx_usbmisc_power_lost_check

commit 21dfa6210c0a80db816e7ece4a39889c3a585fe0 from
git://git.freescale.com/imx/linux-2.6-imx.git

Export imx_usbmisc_power_lost_check.

Signed-off-by: Li Jun <b47624@freescale.com>
---
 drivers/usb/chipidea/usbmisc_imx.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/usb/chipidea/usbmisc_imx.c b/drivers/usb/chipidea/usbmisc_imx.c
index ce02853..947c631 100644
--- a/drivers/usb/chipidea/usbmisc_imx.c
+++ b/drivers/usb/chipidea/usbmisc_imx.c
@@ -420,6 +420,7 @@ int imx_usbmisc_power_lost_check(struct imx_usbmisc_data *data)
 		return 0;
 	return usbmisc->ops->power_lost_check(data);
 }
+EXPORT_SYMBOL_GPL(imx_usbmisc_power_lost_check);
 
 static const struct of_device_id usbmisc_imx_dt_ids[] = {
 	{
-- 
1.7.5.4

